source/game_preparation/calibration/calibration_walk.rst

```diff
@@ -0,0 +1,19 @@
+###############
+Walk
+###############
+
+We need to calibrate the walk speed for each opponent.
+
+************
+Instructions
+************
+
+#.  ``image/home/nao/data/behaviours/body/skills/Walk.py``
+
+    The higher the speed, the more likely the robot is to fall over.
+
+    .. code-block:: bash
+
+        # Range between 0.0 and 1.0. Use 0 in lab and 1 in serious games. v5 robots
+        # do not work with low speed - 0.5 is functional. v6 need further testing.
```

Comment:
```suggestion
        # do not work with low speed - 0.5 is functional. v6 need further testing.
        # lower speeds can be good sometimes, e.g. if the carpet/astroturf is really slippery
        # e.g. in Thailand 2022 a value of 0.4 on v6s was used
        # as any higher and the robots just fell over all the time
        # if the robot falls over in regular walking, then it'll be far slower going from point A to B
```

Reply:
can you put these in the python file and here we can say to look at the python file for more information?